package.json

```diff
@@ -18,8 +18,8 @@
   },
   "dependencies": {
     "async": "1.5.2",
-    "babyparse": "0.4.3",
-    "blueimp-md5": "2.1.0",
+    "babyparse": "0.4.5",
```

Comment:
So I think I asked this, but I'll ask again. `babyparse` is the CSV parsing library, right @krystophv? Which means we use it for CareLink ingestion, so as I dev updating this it'd be good to at least do a smoke test of CareLink uploading as part of developer testing.